Code/GraphMol/FileParsers/CDXMLParser.cpp

```diff
@@ -422,6 +422,20 @@ bool parse_fragment(RWMol &mol, ptree &frag,
       } else if (bond.display == "WedgedHashBegin" ||
                  bond.display == "WedgedHashEnd") {
         bnd->setBondDir(Bond::BondDir::BEGINWEDGE);
+      } else if (bond.display == "Wavy") {
+        switch(bond.getBondType()) {
+            case Bond::BondType::SINGLE:
+              bnd->setBondDir(Bond::BondDir::UNKNOWN);
```

Comment:
This is probably not part of this PR, but the mol file parser sets the `_MolFileBondCfg` property on bonds which have wedging information. This is then used by the CXSMILES writer and in `reapplyMolBlockWedging()`. It's worth considering doing that for the CDXML parser as well at some future point.